### PR TITLE
add Bio:: and Schedule::SGELK dependency install + checks

### PR DESCRIPTION
**Pull request by chrisgulvik**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -169,7 +169,7 @@ clean-raxml:
 # module so that Make can do its job.
 install-perlModules:
 	@echo "Installing Perl modules using cpanminus"
-	for package in Config::Simple File::Slurp Math::Round Number::Range Statistics::Distributions Statistics::Basic Graph::Centrality::Pagerank String::Escape Statistics::LineFit; do \
+	for package in Bio::Perl Bio::FeatureIO Config::Simple File::Slurp Math::Round Number::Range Statistics::Distributions Statistics::Basic Graph::Centrality::Pagerank String::Escape Statistics::LineFit; do \
```
Comment (lskatz) [inline Makefile]: Installation for Bio::Perl and Bio::FeatureIO is cumbersome and so I'll leave this part out.  It is up to the user to install these.

File: scripts/launch_stampy.sh [hunk 2]
```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 
 
-SCRIPT=$(basename $0);
+set -e
```
Comment (lskatz) [inline scripts/launch_stampy.sh]: You sure you want to `set -e`?

File: scripts/launch_stampy.sh [hunk 3]
```diff
@@ -118,30 +119,22 @@ else
...
 fi
 
+
 # Add onto xopts so that files can be overwritten 
 # and so that we don't worry whether XOPTS is an empty variable.
-XOPTS="$XOPTS --overwrite "; 
+XOPTS="$XOPTS --overwrite "
 # Map with stampy
 COMMAND="stampy.py -g $REFERENCE -h $REFERENCE -t $CPUS --substitutionrate=0.01 --minposterior=20 --sensitive --inputformat=fastq -f sam -o $TMP/$OUT_PREFIX.sam --insertsize=500 --insertsd=250 -M $STAMPY_READS $XOPTS"
 eval $COMMAND
```
Comment (lskatz) [inline scripts/launch_stampy.sh]: Unsure why you are removing error checking

**Comment by lskatz** [comment]
I have a few notes that should be addressed before I can accept the change.

File: scripts/launch_stampy.sh (hunk 3 above)
Comment (chrisgulvik) [inline scripts/launch_stampy.sh]: 'set -e' accomplishes this; error checking throughout and cleaner script too

File: scripts/launch_stampy.sh (hunk 2 above)
Comment (chrisgulvik) [inline scripts/launch_stampy.sh]: Yes. It ensures if one system call fails, the script exits and doesn't waste time looking for output that wasn't properly made due to an error.

File: Makefile (hunk 1 above)
Comment (chrisgulvik) [inline Makefile]: That's understandable

**Pull request merged**